orchestra/communication/staffing.py

```diff
@@ -59,17 +59,16 @@ def handle_staffing_response(worker, staffing_request_inquiry_id,
                 'Cannot reject after accepting the task')
 
         response.is_available = is_available
+        response.save()
```

Comment:
curiously, why did we need to add this and didn't need it before?

Reply:
Technically we don't need it. I just added this for clarity.

The reason we didn't have to save it was because if `is_available` is true, then we call `assign_task` or `reassign_task` and those function set `is_available` to true together with `is_winner`.  When `is_available` is false, the default value is already false and there is no need to save it. 

We could remove it since it is doing nothing but the current code is not super clear either.

Reply:
Very much your call, as you're closer to the code. Sometimes saving twice bites you beyond performance reasons if you safe an extraneous field you weren't ready to save yet, but whatever you decide :)

Reply:
OK. I will remove it but will leave a comment for future references.